ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -326,6 +326,7 @@ def create_policy(
         """
         policy = self.trainer.create_policy(parsed_behavior_id, brain_parameters)
         policy.create_tf_graph()
+        policy.initialize_or_load()
```

Comment:
In the case of symmetric games (soccer/tennis), this ensures that both policies are initialized from the checkpoint.